scripts/ci/issue.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+
+# be stricter
+set -eou pipefail
+
+# Use jq to get the outdated libs from the "cargo outdated" JSON string and generate Github issue labels
+#
+# Takes the following function args:
+# $1 - "cargo outdated" JSON string
+#
+# Returns:
+# - The labels that are used when searching for or creating a Github issue, in format 'lib_name-current_ver-latest_ver'
+generate_issue_labels() {
+	local outdated_libs_json=$1
+	if [[ -z "$outdated_libs_json" ]]; then
+		echo "required parameter for generate_issue_labels() is empty" 1>&2
+		exit 1
+	fi
+
+	# Export ISSUE_LABELS in format - "crate_name-current_version-latest_version;..."
+	local issue_labels
+	issue_labels=$(echo "$outdated_libs_json" | jq -r 'select(.crate_name | startswith("uniffi-") or .=="zcash").dependencies[] | select(.project != .latest) | (.name+"-"+.project+"-"+.latest)' | sort -u | tr '\n' ';')
```

Comment:
Please split the long commands one each line like
```suggestion
	issue_labels=$(echo "$outdated_libs_json" |
	        jq -r 'select(.crate_name | startswith("uniffi-") or .=="zcash").dependencies[] | select(.project != .latest) | (.name+"-"+.project+"-"+.latest)' |
	        sort -u |
	        tr '\n' ';')
```